config/config.go

```diff
@@ -388,6 +388,17 @@ type Local struct {
 	// BlockServiceCustomFallbackEndpoints is empty.
 	// The archiver is randomly selected, if none is available, will return StatusNotFound (404).
 	EnableBlockServiceFallbackToArchiver bool `version[16]:"true"`
+
+	// CatchupBlockValidateMode defines the degree of validation used by the catchup service. This field is a bit-field, where the
+	// default enabled checks have their corresponding bit set as zero, and the default disabled checks have their corresponding bit clear.
+	// bit 0, default enable: verify the block certificate
```

Comment:
I think it's confusing that some 1 bits turn on a feature and some 1 bits turn off a feature. If we go with the mspgack default-0 value, can we have the default be 'all checks on' and then any 1 bit turns off a check? Also I'm hoping that 'all checks on' is a good default and these things will mostly be turned off to make some dev and test operations faster.

Reply:
it's not the msgp default here.. it's the json marshaler.
and no - all checks is *not* what we want by default. I think that the descriptions need to be further clarified.